openedge-plugin/src/main/java/org/sonar/plugins/openedge/OpenEdgePlugin.java

```diff
@@ -66,6 +66,7 @@
 
   @Override
   public void define(Context context) {
+    int foo = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "foo". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1854)